lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -0,0 +1,346 @@
+<?php
+/**
+ * Manage and give access to lizmap configuration.
+ *
+ * @author    3liz
+ * @copyright 2012 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+
+use Lizmap\Project;
+
+/**
+ * @deprecated
+ * @FIXME getXml, getComposer
+ * Verify this methods are not used in external modules so we can delete them without risk, otherwise, we have to implement them
+ * in Project and call it here
+ */
+class lizmapProject
+{
+    /**
+     * @var Project\Project
+     */
+    protected $proj;
+
+    /**
+     * constructor.
+     *
+     * @param string           $key      : the project name
+     * @param LizmapRepository $rep      : the repository
+     * @param mixed            $context
+     * @param mixed            $services
+     */
+    public function __construct($key, $rep, $context, $services)
```

Comment:
wrong signature, it should use the same parameters as the original class. The goal of this class is to not break modules that are still using lizmapProject